diff --git a/app/serializers/rest/media_attachment_serializer.rb b/app/serializers/rest/media_attachment_serializer.rb
--- app/serializers/rest/media_attachment_serializer.rb
+++ app/serializers/rest/media_attachment_serializer.rb
@@ -11,29 +11,31 @@
     object.id.to_s
   end

   def url
-    if object.not_processed?
-      nil
-    elsif object.needs_redownload?
-      media_proxy_url(object.id, :original)
-    else
-      full_asset_url(object.file.url(:original))
-    end
+    # if object.not_processed?
+    #   nil
+    # elsif object.needs_redownload?
+    #   media_proxy_url(object.id, :original)
+    # else
+    #   full_asset_url(object.file.url(:original))
+    # end
+    object.remote_url.presence
   end

   def remote_url
     object.remote_url.presence
   end

   def preview_url
-    if object.needs_redownload?
-      media_proxy_url(object.id, :small)
-    elsif object.thumbnail.present?
-      full_asset_url(object.thumbnail.url(:original))
-    elsif object.file.styles.key?(:small)
-      full_asset_url(object.file.url(:small))
-    end
+    # if object.needs_redownload?
+    #   media_proxy_url(object.id, :small)
+    # elsif object.thumbnail.present?
+    #   full_asset_url(object.thumbnail.url(:original))
+    # elsif object.file.styles.key?(:small)
+    #   full_asset_url(object.file.url(:small))
+    # end
+    object.thumbnail_remote_url.presence
   end

   def preview_remote_url
     object.thumbnail_remote_url.presence
